faw/pdf-observatory/faw_analysis_set_parse.py

```diff
@@ -35,6 +34,13 @@ def _import_from_path(mod_name, pth):
 pdf_etl_parse = _import_from_path('pdf_etl_parse', '../pdf-etl-parse/main.py')
 
 COL_NAME = 'as_parse'
+"""Name of parse queue collection."""
+MIN_REQUIRED_PARSE_SPACE_BYTES = 2 * 1024 * 1024  # 2 MB
```

Comment:
For now I set a 2MB minimum; this seems like overkill for any sane parsing case I can think of, but maybe we want to go higher since this could be cutting it close for comfortable ssh access.

Reply:
We should probably start with 2GB